demo/config.js

```diff
@@ -0,0 +1,15 @@
+//Config file
+//TODO: Logic should be separated from configuration
+
+var config_data = {
```

Comment:
This will become a global when not declared within a (limiting) scope. Try to wrap these types of initialization in an [iife](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression).